<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecDialog</class>
 <widget class="QDialog" name="SecDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{

	
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(166, 204, 255, 255), stop:1 rgba(255, 255, 255, 255));

}</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>481</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Linear Regression</string>
   </property>
  </widget>
  <widget class="QLabel" name="numPoints">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	qproperty-alignment:  'AlignVCenter | AlignLeft';
	border:  1px solid gray;
	background-color: white
}


</string>
   </property>
   <property name="text">
    <string>Number of Points</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPoint">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	qproperty-alignment:  'AlignVCenter | AlignCenter ';
	border:  1px solid gray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="xVal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	qproperty-alignment:  'AlignVCenter | AlignLeft';
	border:  1px solid gray;
	background-color: white
}


</string>
   </property>
   <property name="text">
    <string>X Coordinates</string>
   </property>
  </widget>
  <widget class="QLabel" name="yVal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	qproperty-alignment:  'AlignVCenter | AlignLeft';
	border:  1px solid gray;
	background-color: white
}


</string>
   </property>
   <property name="text">
    <string>Y Coordinates</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditX">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	qproperty-alignment:  'AlignVCenter | AlignLeft ';
	border:  1px solid gray;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditY">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	qproperty-alignment:  'AlignVCenter | AlignLeft ';
	border:  1px solid gray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border:  1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);

}</string>
   </property>
   <property name="text">
    <string>Calculate </string>
   </property>
  </widget>
  <widget class="QPushButton" name="Clear">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border:  1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);

}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
